### ISSUE TITLE
[Code Style] Indentation rules

### ISSUE THREAD

**Issue by kaijchen** [apache/incubator-uniffle#54]
There are 2 indentation styles in the current code base. Which are mostly the same except for line wrapping.
It would be better to have it unified early. How do you think?

### 4 spaces for line wrapping

```java
public static void main(String[] args)
    throws Exception {
  for (int i = 0; i < args.length; i++) {
    System.out.println("Greeting the " + i +
        "-th argument");
    System.out.println("hello, " + args[i]);
  }
}
```

### 2 spaces for line wrapping

```java
public static void main(String[] args)
  throws Exception {
  for (int i = 0; i < args.length; i++) {
    System.out.println("Greeting the " + i +
      "-th argument");
    System.out.println("hello, " + args[i]);
  }
}
```

**Comment by kaijchen** [comment]
Personally I prefer the "4 spaces" way, because it differentiates line wrapping with block indents.

**Comment by jerryshao** [comment]
I'm +1 on 4 spaces for line wrapping.

**Comment by kaijchen** [comment]
Closed for #56.
